components/molecule/select/README.md

```diff
@@ -0,0 +1,152 @@
+# MoleculeSelect
+
+`MoleculeSelect` is a customized `select` created from a combination of `AtomInput`, `MoleculeInputTags`, `MoleculeDropdownList` and `MoleculeDropdownOption`
+
+It allows Single and Multiple Selection
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-molecule-select --save
+```
+
+## Usage
+
+```js
+import MoleculeSelect, {
+  moleculeSelectDropdownListSizes,
+  MoleculeSelectOption
```

Comment:
this shouldn't be working anymore, right?

it should be imported from '@s-ui/react-molecule-dropdown-option'